packages/api-generator/src/locale/en/VOtpInput.json

```diff
@@ -10,5 +14,10 @@
     "finish": "Emitted when the input is filled completely and cursor is blurred.",
     "focus": "Emitted when component is focused.",
     "keydown": "Emitted when **any** key is pressed."
+  },
+  "exposed": {
+    "blur": "Blurs the input.",
+    "focus": "Focuses the input.",
+    "reset": "Resets the input's value."
```

Comment:
I'm working on making these inherit properly.

Reply:
Was this ever added?